chia/pools/pool_wallet.py

```diff
@@ -350,12 +350,16 @@ async def create(
             if spend.coin.name() == launcher_coin_id:
                 launcher_spend = spend
         assert launcher_spend is not None
-        await self.wallet_state_manager.pool_store.add_spend(self.wallet_id, launcher_spend, block_height)
+        await self.wallet_state_manager.pool_store.add_spend(
+            self.wallet_id, launcher_spend, block_height, in_transaction
```

Comment:
PoolWallet.create is called only from within `new_coin_state` and it's using `in_transction=False` so that seems wrong

Reply:
NVm i saw that it's been changed to True